internal/testrunner/runners/system/tester.go

```diff
@@ -120,6 +120,15 @@ var (
 						regexp.MustCompile(`Component state changed .* \(HEALTHY->DEGRADED\): Degraded: pid .* missed .* check-in`),
 					},
 				},
+				{
+					// HTTPJSON template error.
+					includes: regexp.MustCompile(`^error processing response: template: :\d+:\d+: executing "" at <`),
```

Comment:
Would it be possible to have a test case that reproduces this issue?

Reply:
How is this done?

Reply:
We could maybe add a [false positives](https://github.com/elastic/elastic-package/tree/f61e253f3a81a5b9f10bb377eb95efad87fd2507/test/packages/false_positives) test case.

Reply:
The current HTTPJSON false positive case is already hitting this logic.